### PR TITLE
allowing templates to be applied AFTER the cluster is fully up

### LINKED ISSUE #180: allow for manifests to be applied after the cluster is fully up

### PR DESCRIPTION
**Pull request by aleiner**
## What type of PR is this?

_(REQUIRED)_

- [ ] bug
- [ ] cleanup
- [ ] documentation
- [x] feature

## What this PR does / why we need it:

_(REQUIRED)_

Allows for manifest files to be applied after the cluster is up.  Current functionality only allows for manifests to be applied after the first server node is up.

## Which issue(s) this PR fixes:

_(REQUIRED)_

Fixes #180

### CODE REVIEW

File: roles/cluster_manifest/vars/main.yml
Comment (mddamato) [inline roles/cluster_manifest/vars/main.yml:1]: delete blank file?

File: roles/cluster_manifest/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+kubernetes_api_server_host: "{{ hostvars[groups['rke2_servers'][0]].inventory_hostname }}"
```
Comment (mddamato) [inline roles/cluster_manifest/defaults/main.yml:2]: unused var?

**Review by mddamato — CHANGES_REQUESTED** [review]

File: roles/cluster_manifest/vars/main.yml
Comment (aleiner) [reply roles/cluster_manifest/vars/main.yml:1]: deleted

**Review by mddamato — APPROVED** [review]

**Comment by jcox10** [comment]
How is this different than the `add-manifest-addons.yml`? It should be at least documented in `inventory/sample/group_vars/rke2_servers.yml` at the _bare_ minimum.

**Comment by aleiner** [comment]
@jcox10 - Thanks for keeping us honest. PR has been submitted to address the additional documentation.

We will rework some of the logic before a new release where the variable names may change to make things more streamlined.

**Comment by mddamato** [comment]
@jcox10 You're correct. please see https://github.com/rancherfederal/rke2-ansible/issues/198

**Pull request merged**